src/js/constants/blacklistServiceConfig.js

```diff
@@ -0,0 +1,16 @@
+let BlackListServiceConfig = [
```

Comment:
Let's rename this file and provide the black list as a variable so that it is consistent with the other constant files.  How about `ServiceConfig.BLACK_LIST`?